### ISSUE TITLE
Chapter 11 Question 8.3 cross-validation code

### ISSUE THREAD

**Issue by GorkemP** [ageron/handson-ml#321]
Hi Aurelion,

In question 8.3, there must be a cross-validation. Indeed, in the code you have provided you use RandomizedSearchCV in cell 122. In the output, it says that there will be 3 folds, but I cannot find where we have setted this 3 fold thing.

It seems to me that there is only one fold and we have already gave validation set to the classifier.

```python
rnd_search = RandomizedSearchCV(DNNClassifier(random_state=42), param_distribs, n_iter=50,
                                random_state=42, verbose=2)
rnd_search.fit(X_train1, y_train1, X_valid=X_valid1, y_valid=y_valid1, n_epochs=1000)
```

**Comment by willer94** [comment]
by default, it is 3 fold, you can set cv=5 for 5 folder in RandomizedSearchCV(),  if you want. refer to sscikit-learn user guide.

**Comment by ageron** [comment]
Hi @GorkemP ,
As @willer94 pointed out (thanks), the default number of folds is cv=3.
Hope this helps.

**Comment by GorkemP** [comment]
Yes, definetely. My mistake to not to look for default values. Just a reminder, I checked the documentation of RandomizedSearchCV() and default value of None is planned to be 5-fold cross validation in version 0.22.
[Scikit-learn RandomizedSearchCV](https://scikit-learn.org/stable/modules/generated/sklearn.model_selection.RandomizedSearchCV.html)
